### DISCUSSION TITLE
How to use bearer authentication?

### DISCUSSION THREAD

**Discussion by marvix** [fastify/fastify#3171] [Q&A] [answered]
I could not find any example or tutorial about how to implement bearer auth with Fastify. 
Would anyone please provide a working example?

**Comment by mcollina** [comment, marked as answer]
Check out https://github.com/fastify/fastify-bearer-auth

**Reply by marvix** [reply]
I checked it before, but the problem no much information there, such as how to generate the token and how to verify it.

**Reply by jsumners** [reply]
What is unclear about the documentation?
